schemas/context/environment.schema.json

```diff
@@ -32,6 +32,12 @@
           "description":
             "The browser specific details such as browser name, version, javascript version, user agent string, accept language."
         },
+        "xdm:viewabilityDetails": {
```

Comment:
I don't think the name `Details` carries too much information or clarifies the intent vs. `xdm:viewability`. Maybe `xdm:adViewability` makes it clear enough.

Reply:
Also, from the examples it is not entirely clear what ad unit this refers to.

Reply:
Changed name of schema.

As far as what ad unit, a viewability schema will be tied to a advertisement schema with PR 282. A single experience event should only describe a single ad, thus the ad is implied by its' other contexts.